cmake/nokia_ngage.cmake

```diff
@@ -0,0 +1,133 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
This file should be integrated into `CMakeLists.txt`, but once you get ci running for the currently supported SDL3 platforms, I'll try to do this.

Reply:
https://github.com/ngagesdk/ngage-toolchain/issues/13

Apparently you tried this before. :)

Reply:
At least native (Windows) CI is now working: https://github.com/ngagesdk/ngage-toolchain/actions/workflows/nokia-ngage.yml

Reply:
This file can now be removed.